app/backend/Dockerfile

```
@@ -0,0 +1,22 @@
+FROM python:3.11-slim
+
+WORKDIR /app
+
+# Install Poetry
+RUN pip install poetry==1.7.1
```

Hadolint (2.14.0)

[warning] Avoid use of cache directory with pip. Use `pip install --no-cache-dir <package>`

(DL3042)